django_ipam/templates/admin/django-ipam/change_form.html

```diff
@@ -0,0 +1,52 @@
+{% extends "admin/change_form.html" %}
+{% load i18n admin_urls %}
+{% load static %}
+
+{% block after_related_objects %}
+    <li style="list-style-type:none">
+      <div id="graph">
+      </div>
+      <div style="padding:10px">
```

Comment:
we should really use a CSS file instead of inline CSS, this applies to all the other inline styling as well